packages/xod-client/src/hinting/utils.js

```diff
@@ -0,0 +1,66 @@
+import * as R from 'ramda';
```

Comment:
Please, no more modules with a name `utils`/`helpers`/`tools` and other “one size fits all”. Name it after the function it carries (`errorCollectors`?).